src/tests/roc_core/test_random.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2015 Roc Streaming authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <CppUTest/TestHarness.h>
+
+#include "roc_core/stddefs.h"
+#include "roc_core/secure_random.h"
+
+namespace roc {
+namespace core {
+
+TEST_GROUP(random) {
+};
+
+TEST(random, min_min) {
+    uint32_t lo_hi = 0;
+    uint32_t res;
+    bool ok = secure_random(lo_hi, lo_hi, res);
+
+    CHECK(ok);
+    LONGS_EQUAL(res, lo_hi);
+}
```

Comment:
Could you please add the same check for `fast_random` to each test?

Reply:
done